lib/src/center.dart

```diff
@@ -1,5 +1,32 @@
+import 'package:turf/bbox.dart';
 import 'package:turf/helpers.dart';
 
-Position center(BBox bBox) {
-  return Position.named(lat: (bBox.lat1 + bBox.lat2) / 2, lng: (bBox.lng1 + bBox.lng2) / 2);
+/// Takes a {@link Feature} or {@link FeatureCollection} and returns the absolute center point of all features.
+///
+/// @name center
```

Comment:
You have imported the JSDoc comments here, but in Dart, there is a little different syntax, basically you don't have annotations such as `@name ...` or `@param {...}`, you simply describe what the function does in a sentence and wrap `[ ]` those brackets around a variable, function, or type name.

Like this:
```dart
/// Takes a [Feature] or [FeatureCollection] and returns the absolute center point of all features.
///
```

You can find a documentation on documentation 😄 here: https://dart.dev/guides/language/effective-dart/documentation